spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/partials/EvaluateIntoErrorMessage.scala

```diff
@@ -18,18 +18,30 @@ package za.co.absa.spark.commons.errorhandling.partials
 
 import org.apache.spark.sql.Column
 import org.apache.spark.sql.functions.struct
-import za.co.absa.spark.commons.errorhandling.ErrorMessageSubmit
+import za.co.absa.spark.commons.errorhandling.{ErrorMessage, ErrorMessageSubmit}
```

Comment:
```suggestion
import za.co.absa.spark.commons.errorhandling.ErrorMessageSubmit
```
Removed import not used in code.